vendor/github.com/containers/common/pkg/config/config.go

```diff
@@ -427,6 +427,10 @@ type EngineConfig struct {
 	// ServiceDestinations mapped by service Names
 	ServiceDestinations map[string]Destination `toml:"service_destinations,omitempty"`
 
+	Farms map[string][]string `toml:"farms,omitempty"`
```

Comment:
I don't see a PR for that.

Reply:
I haven't created one yet. I was planning on doing that once this PR gets some reviews and head nods on it. 
Just created on https://github.com/containers/common/pull/1586

Reply:
Thanks!